WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanViewModel.kt

```diff
@@ -262,7 +291,41 @@ class ScanViewModel @Inject constructor(
         )
     )
 
-    sealed class UiState { // TODO: ashiagr add states for loading, error as needed
-        data class Content(val items: List<JetpackListItemState>) : UiState()
+    sealed class UiState {
+        object FullScreenLoadingUiState : UiState() {
+            @DrawableRes val image = R.drawable.img_illustration_empty_results_216dp
```

Comment:
🔍 Nitpick: The illustration looks a bit weird on "loading" screen. I'd consider showing a progress instead of the "result empty" illustration. Wdyt?

Reply:
I'll add a skeleton with a Shimmer effect.

Reply:
As discussed, ended up using progress bar in d9ea5d2